end-to-end/ratelimit-service/ratelimit.proto

```diff
@@ -1,4 +1,4 @@
-// FROM https://github.com/envoyproxy/envoy/blob/master/source/common/ratelimit/ratelimit.proto
```

Comment:
Can't the end-to-end test literally compile from ../../common/ratelimit/ratelimit.proto?

Reply:
TBH, I kind of like to have it all packaged in the same directory, since the example does not depend on other project files and can easily be moved around or extracted to it's own "samples" repo.

It would work if running node locally, but for the Docker image we would need to add an `ADD` instruction to this file, perhaps from the github URL.